### PR TITLE
Tutorial: Using api metrics with prometheus

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
https://github.com/crate/cloud/issues/1151

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
https://github.com/crate/cloud-tutorials/pull/110 must be merged first

File: docs/cloud-monitoring.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
+.. _cloud-monitoring:
+
+Cloud Monitoring
+================
+
+This tutorial shows a way to monitor your CrateDB Cloud clusters using the
```
Comment (kojinkai) [inline docs/cloud-monitoring.rst]:
```suggestion
This tutorial demonstrates how you can monitor your CrateDB Cloud
```

File: docs/cloud-monitoring.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+There should be an endpoint with your Organization ID with state ``UP``. This
+means that Prometheus is able to connect to the API and is scraping the
+available metrics.
+
+Grafana
+-------
+
+Grafana doesn't need any special configuration. You can run it either in a
+Docker container or as a local installation, it doesn't matter in this use
```
Comment (kojinkai) [inline docs/cloud-monitoring.rst]:
```suggestion
Docker container or as a local installation, it doesn't matter for this use
```

**Review by kojinkai — APPROVED** [review]

File: docs/cloud-monitoring.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
+.. _cloud-monitoring:
+
+Cloud Monitoring
+================
+
+This tutorial demonstrates how you can monitor your CrateDB Cloud cluster
+using API metrics.
```
Comment (SStorm) [inline docs/cloud-monitoring.rst]:
```suggestion
using the exposed Prometheus metrics.
```

File: docs/cloud-monitoring.rst [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+The visualization tool `Grafana`_ is used with `Prometheus`_ to
+scrape the API endpoints for the relevant metrics and visualize them. The
```
Comment (SStorm) [inline docs/cloud-monitoring.rst]:
```suggestion
scrape the API endpoint that exposes metrics and visualize them. The
```

File: docs/cloud-monitoring.rst [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+      static_configs:
+        - targets: ["console.cratedb.cloud"]
+
+Substitute the ``ORGID`` with the ID of your organization. It can be found in
+the Settings page in the CrateDB Cloud Console:
+
+.. image:: /_assets/img/cloud-monitoring-org-id.png
+   :alt: Organization ID
+
+API Key and Secret can be generated in your `account page`_ in the CrateDB
```
Comment (SStorm) [inline docs/cloud-monitoring.rst]:
```suggestion
An API Key and Secret can be generated in your `account page`_ in the CrateDB
```

File: docs/cloud-monitoring.rst [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+By default, Grafana is exposed on port ``3000``. Go to
+``http://localhost:3000/`` to access it.
+
+Data source
+'''''''''''
+
+Now you can add Prometheus as a data source in Grafana under ``Configuration
+-> Data sources``. Choose Prometheus, use ``http://localhost:9090/`` as the
+URL, and leave the rest on default:
```
Comment (SStorm) [inline docs/cloud-monitoring.rst]:
```suggestion
URL, and leave the rest as default:
```

File: docs/cloud-monitoring.rst [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    - Average query duration along with the type of query
+    - Queries per second along with the type of query
+
+
+.. _account page: https://crate.io/docs/cloud/reference/en/latest/api.html
+.. _Cloud Console: https://console.cratedb.cloud/?utm_campaign=2022-Q3-WS-Developer-Motion&utm_source=docs
+.. _Grafana: https://grafana.com/
+.. _Grafana documentation: https://grafana.com/docs/grafana/latest/setup-grafana/installation/
+.. _Prometheus: https://grafana.com/oss/prometheus/
+.. _this snippet: https://raw.githubusercontent.com/crate/cloud-tutorials/master/docs/_extra/cratedb-cloud-cluster-dashboard.json
```
Comment (SStorm) [inline docs/cloud-monitoring.rst:159]: This is currently a 404, do you need to merge something?

**Review by SStorm — APPROVED** [review]
Looks good, just a few wording suggestions :)

File: docs/cloud-monitoring.rst (hunk 7 above)
Comment (matkuliak) [reply docs/cloud-monitoring.rst:159]: Yes, the https://github.com/crate/cloud-tutorials/pull/110 dashboard is a separate PR.

**Pull request merged**